roles/dcos_agent_windows/tasks/dcos_upgrade.yml

```diff
@@ -0,0 +1,26 @@
+- name: Upgrade | Download upgrade_win.ps1
+  win_get_url:
+    url: "{{ dcos['config']['bootstrap_url'] }}/{{ dcos_version_specifier }}/genconf/serve/windows/upgrade_from_{{ dcos['version_to_upgrade_from'] }}/latest/dcos_node_upgrade.ps1"
+    dest: "{{ base_windows_dir }}\\upgrade_win.ps1"
+    force: yes
```

Comment:
what is the reason of not applying specific version identifiers to the path like in https://github.com/dcos/dcos-ansible/blob/master/roles/dcos_agent/tasks/dcos_upgrade.yml ?

Reply:
There is no need to use specific version identifiers because winpanda doesn't work with them (we don't use symlink as the path like in linux-node)

Reply:
would something not working if we use the same method as on linux?

If not there is no reason why we should do something different. Having these identifiers makes debugging upgrade issue drastically easier as you can see right away those specifiers.

What winpanda does under the hood doesn't matter for dcos-ansible